packages/wallet/src/redux/protocols/indirect-funding/__tests__/scenarios.ts

```diff
@@ -36,7 +37,7 @@ const ledger4 = ledgerCommitment({ turnNum: 4, balances: oneThree });
 const ledger5 = ledgerCommitment({ turnNum: 5, balances: oneThree });
 const app0 = appCommitment({ turnNum: 0, balances: oneThree });
 const app1 = appCommitment({ turnNum: 1, balances: oneThree });
-const existingLedgerFundingSharedData = setChannels(existingLedgerPreSuccess.sharedData, [
+const existingLedgerFundingSharedData = setChannels(newLedgerSuccess.store, [
```

Comment:
This line seems strange. Should the existingLedgerFunding preSucces instead use the newLedgerSuccess store to generate its store?

Reply:
Hmm, I think I'd lean towards leaving it this way since relying on `existingLedgerPreSuccess` seems less accurate.

We're trying to set up the `sharedData` before the existing ledger protocol runs but by using `existingLedgerPreSuccess` we're using `sharedData` of the existing ledger channel protocol that is partially completed.

This seems to be closer to what actually happens, since the existing ledger channel protocol runs after the new ledger channel protocol has succeeded in the past.